web/templates/modules/PageRate/PageRateWidgetModule.tpl

```diff
@@ -1,7 +1,8 @@
 {strip}
+{if $enabled == true}
 {if $type=="P" || $type=="M" || $type=="Z" || $type=="X"}
```

Comment:
The contents of this rating module (within `{if $enabled == true}`) should be indented.